.php_cs

```diff
@@ -0,0 +1,19 @@
+<?php
+
+$finder = Symfony\CS\Finder\DefaultFinder::create()
+    ->in(array(__DIR__))
+;
+
+return Symfony\CS\Config\Config::create()
+    ->level(Symfony\CS\FixerInterface::SYMFONY_LEVEL)
+    ->fixers(array(
+        '-unalign_double_arrow',
```

Comment:
Symfony decided a while ago to unalign everything. I guess good bundles should follow that practice too.